pkg/combustion/templates/15-rke2-single-node-installer.sh.tpl

```diff
@@ -2,13 +2,18 @@
 set -euo pipefail
 
 mount /var
+
 mkdir -p /var/lib/rancher/rke2/agent/images/
 cp {{ .imagesPath }}/* /var/lib/rancher/rke2/agent/images/
 
 {{- if .vipManifest }}
 mkdir -p /var/lib/rancher/rke2/server/manifests/
 cp {{ .vipManifest }} /var/lib/rancher/rke2/server/manifests/{{ .vipManifest }}
 {{- end }}
+{{ if .manifestsPath }}
+mkdir -p /var/lib/rancher/rke2/server/manifests/
+cp {{ .manifestsPath }}/* /var/lib/rancher/rke2/server/manifests/
+{{ end }}
```

Comment:
Nit:
```suggestion
{{- if .manifestsPath }}
mkdir -p /var/lib/rancher/rke2/server/manifests/
cp {{ .manifestsPath }}/* /var/lib/rancher/rke2/server/manifests/
{{- end }}
```

I'd also put empty lines before and after this conditional block. This way the template becomes more readable.